templates/quickstart-bitbucket-dc.template.yaml

```diff
@@ -729,32 +729,45 @@ Resources:
             Version: 2012-10-17
             Statement:
               - Action:
+                  - rds:AddTagsToResource
                   - ec2:DetachVolume
                   - ec2:AttachVolume
+                  - rds:DescribeDBSnapshots
+                  - rds:CopyDBSnapshot
                   - ec2:CreateTags
-                  - ec2:CreateSnapshot
+                  - rds:DeleteDBSnapshot
                   - ec2:CreateVolume
+                  - rds:PromoteReadReplica
+                  - rds:CreateDBSnapshot
+                  - rds:RestoreDBInstanceFromDBSnapshot
+                  - autoscaling:CreateOrUpdateTags
+                  - ec2:CreateSnapshot
+                  - rds:DescribeDBInstances
+                  - rds:ModifyDBInstance
                 Resource:
-                  - !Sub "arn:${AWS::Partition}:ec2:*:${AWS::AccountId}:instance/*"
-                  - !Sub "arn:${AWS::Partition}:ec2:*:${AWS::AccountId}:volume/*"
+                  - arn:aws:rds:*:887764444972:snapshot:*
+                  - arn:aws:ec2:*:887764444972:instance/*
+                  - arn:aws:ec2:*:887764444972:volume/*
+                  - arn:aws:autoscaling:*:887764444972:autoScalingGroup:*:autoScalingGroupName/*
```

Comment:
We cannot include specific AWS account (`887764444972`) in the general AWS quickstart.

Can we change the account to `*` as well?

Reply:
resolved